src/ToggleBadge.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { theme as themeGet, space } from 'styled-system'
+import theme from './theme'
+
+const ToggleBadge = styled.div`
+  border-radius: 2px;
+  display: inline-block;
+  font-size: ${props => props.theme.fontSizes[0]}px;
+  font-weight: 600;
+  background-color: ${props =>
+    props.selected ? props.theme.colors.lightBlue : props.theme.colors.white};
+  color: ${props => props.theme.colors.darkBlue};
+  ${space};
+  &:hover {
+    background-color: ${props => props.theme.colors.lightBlue};
+  }
+`
+
+ToggleBadge.displayName = 'ToggleBadge'
+
+const numberStringOrArray = PropTypes.oneOfType([
+  PropTypes.number,
+  PropTypes.string,
+  PropTypes.array
+])
+
+ToggleBadge.propTypes = {
+  selected: PropTypes.bool,
+  px: numberStringOrArray,
```

Comment:
I think you might be able to simplify the space propTypes as `...space.propTypes`

https://github.com/jxnblk/styled-system/blob/master/docs/api.md#proptypes